src/open_inwoner/qmatic/client.py

```diff
@@ -0,0 +1,141 @@
+from datetime import datetime
+
+from ape_pie.client import APIClient
+from pydantic import BaseModel
+from zgw_consumers.client import build_client
+from zgw_consumers.models import Service
+
+from open_inwoner.utils.api import JSONEncoderMixin
+
+from .exceptions import QmaticException
+from .models import QmaticConfig
+
+# API DATA DEFINITIONS
+
+
+class ServiceDict(BaseModel):
```

Comment:
If these aren't TypedDict's anymore the name doesn't need the Dict postfix :wink: